lib/platform/gitlab/index.spec.ts

```diff
@@ -69,13 +72,17 @@ describe('platform/gitlab', () => {
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
     it(`should accept custom endpoint`, async () => {
-      httpMock.scope('https://gitlab.renovatebot.com').get('/user').reply(200, {
+      const endpoint = 'https://gitlab.renovatebot.com';
+      httpMock.scope(endpoint).get('/user').reply(200, {
         email: 'a@b.com',
         name: 'Renovate Bot',
       });
+      httpMock.scope(endpoint).get('/version').reply(200, {
+        version: '13.3.6-ee',
+      });
```

Comment:
```suggestion
      })
      .get('/version').reply(200, {
        version: '13.3.6-ee',
      });
```

This can be chained. 🙃 Maybe needs `prettier-fix` run after